gloo/transport/ibverbs/device.cc

```diff
@@ -28,51 +26,18 @@ namespace ibverbs {
 
 static const std::chrono::seconds kTimeoutDefault = std::chrono::seconds(30);
 
-// Scope guard for ibverbs device list.
-class ibv_devices {
- public:
-  ibv_devices() {
-    list_ = ibv_get_device_list(&size_);
-    if (list_ == nullptr) {
-      size_ = 0;
-    }
-  }
-
-  ~ibv_devices() {
-    if (list_ != nullptr) {
-      ibv_free_device_list(list_);
-    }
-  }
-
-  int size() {
-    return size_;
-  }
-
-  struct ibv_device*& operator[](int i) {
-    return list_[i];
-  }
-
- protected:
-  int size_;
-  struct ibv_device** list_;
-};
-
```

Comment:
Can you leave this one in place? It's just a RAII helper, doesn't need to be in the header.